app/components/Home/Yaroslav_Grushchak/Projects/Project_1/components/portfolio-category-buttons/portfolio-category-buttons.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const portfolioCategories = [
+    'all',
+    'panoramas',
+    'portraits',
+    'macro',
+    'journal'
+];
+
+const PotrfolioCategoryButtons = (props) => (
+    <nav className="tab-menu trigger-victim">
+
+        {portfolioCategories.map((item)=> (
+            <li key={item}><a onClick={props.update.bind(this)}>{item}</a></li>
```

Comment:
use arrow functions instead of bind
props.update.bind(this)
->
() => props.update()

Reply:
Done.